### DISCUSSION TITLE
pjax开启后的问题

### DISCUSSION THREAD

**Discussion by zizhaolaisuimuqu** [YunYouJun/hexo-theme-starry#199] [Q&A 疑难杂症]
1. 我将yun._config.yml的pjax项的enable的值改为true的时候（我在yun.config插件那部分的aplayer添加了单曲），在博客里面在x页面进入下一个y页面的时候，x页面的歌曲会继续播放，但是y页面的歌曲也会加载播放。这样子导致在同一个页面有两首歌在不同的位置多次播放。
2. 我用edge（pc）浏览器看的时候发现pjax似乎没有效果，但是用fire fox的时候pjax又有效果。有没有人遇到跟我同样问题的……

（我是小白，大神们尽量讲的详细点儿……(～￣▽￣)～）

**Comment by YunYouJun** [comment]
不如放弃 PJAX？（播放歌曲应该是正常的表现，需要自己关掉某个。以及根据 Chrome 浏览器等规范，歌曲应当是不能默认播放的。
